src/test/java/com/epam/tc/hw5/steps/ex1/DifferentElementsStepsDefinition.java

```diff
@@ -0,0 +1,111 @@
+package com.epam.tc.hw5.steps.ex1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw5.common.DifferentElementsLog;
+import com.epam.tc.hw5.common.DifferentElementsMainContent;
+import com.epam.tc.hw5.common.HomePage;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+
+public class DifferentElementsStepsDefinition {
+    WebDriver driver;
+
+    @Before
+    public void webDriverSetUp() {
+        WebDriverManager.chromedriver().setup();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--start-maximized");
+
+        driver = new ChromeDriver(options);
+    }
+
+
+    @Given("I open home page")
```

Comment:
It's better to separate hooks and steps keeping them in different classes. Hooks will be used by default by Cucumber runner (if the path to hooks is set in glue)